larray/core/session.py

```diff
@@ -154,6 +200,19 @@ def load(self, fname, names=None, engine='auto', display=False, **kwargs):
         display : bool, optional
             whether or not to display which file is being worked on. Defaults
             to False.
+            
+        Examples
+        --------
+        In one module 
+        
+        >>> arr1, arr2, arr3 = ndtest((2, 2)), ndtest(4), ndtest((3, 2))  # doctest: +SKIP
+        >>> Session([('arr1', arr1), ('arr2', arr2), ('arr3', arr3)]).save('input.h5')     # doctest: +SKIP
```

Comment:
wouldn't doing this in two steps (1: create session, 2: save) make things clearer?